src/_common/button/button.styl

```diff
@@ -58,20 +45,27 @@ make-button-size(size)
 	// Hover state is always the same.
 	&:hover
 	&.-hover
-		change-bg('bi-bg', true)
-		theme-prop('border-color', 'bi-bg', true)
+		change-bg(
+			'bi-bg',
+			true
+		)
+		theme-prop(
+			'border-color',
+			'bi-bg',
+			true
+		)
```

Comment:
Any idea what changed to cause this to happen in our `.styl` files now?

Reply:
upgraded stylus recently, maybe it's that. or just the stylus extension we used updated